specification/cognitiveservices/ContentSafety/routes.tsp

```diff
@@ -0,0 +1,91 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-client-generator-core";
+import "@typespec/http";
+import "@typespec/rest";
+import "./models.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+using Azure.ClientGenerator.Core;
+
+namespace ContentSafety;
+
+interface TextOperations {
+  @summary("Analyze Text")
+  @doc("A sync API for harmful content analysis for text. Currently, we support four categories: Hate, SelfHarm, Sexual, Violence.")
+  @convenientAPI
```

Comment:
@mengaims you should remove @convenientAPI it's not necessary anymore and might be causing some of the .NET issues you've been seeing.

Reply:
Removed.